packages/opentelemetry-node/src/NodeTracerProvider.ts

```diff
@@ -37,4 +45,12 @@ export class NodeTracerProvider extends BasicTracerProvider {
   stop() {
     this._pluginLoader.unload();
   }
+
+  register(config: SDKRegistrationConfig = {}) {
+    if (config.contextManager === undefined) {
+      config.contextManager = new AsyncHooksScopeManager();
```

Comment:
should we do `config.contextManager.enable()` ?

Reply:
Do we want to run `enable` on the context manager which is passed in, or just the default one we create ourselves? I don't want to cause odd side-effects if we call `enable` when a user is not expecting us to. The default fallback is probably the most common use-case, and passing in an explicit scope manager is advanced use.

Reply:
https://github.com/open-telemetry/opentelemetry-js/pull/837/commits/faadc39f7666465f03ee9f165b6656d2bcdd2a1a

Reply:
see also https://github.com/open-telemetry/opentelemetry-js/issues/758

Reply:
This at least fixes the inconsistency mentioned in #758